docs/prow.md

```diff
@@ -0,0 +1,397 @@
+| integration-e2e-parallel | `/test integration-e2e-parallel` | No | No | Int (uksouth) |
+| stage-e2e-parallel | `/test stage-e2e-parallel` | No | No | Stage (uksouth) |
+| prod-e2e-parallel | `/test prod-e2e-parallel` | No | No | Prod (uksouth) |
+
+#### Images
+
+| Property | Value |
+|----------|-------|
+| **Job Names** | [`pull-ci-Azure-ARO-HCP-main-images`](https://prow.ci.openshift.org/?job=pull-ci-Azure-ARO-HCP-main-images)<br>[`pull-ci-Azure-ARO-HCP-main-image-updater-images`](https://prow.ci.openshift.org/?job=pull-ci-Azure-ARO-HCP-main-image-updater-images)<br>[`pull-ci-Azure-ARO-HCP-main-periodic-images`](https://prow.ci.openshift.org/?job=pull-ci-Azure-ARO-HCP-main-periodic-images) |
+| **Triggers** | `/test images`<br>`/test image-updater-images`<br>`/test periodic-images` |
+| **Status** | Always runs (required) |
+| **Step Registry** | [images](https://steps.ci.openshift.org/job?org=Azure&repo=ARO-HCP&branch=main&test=images)<br>[image-updater-images](https://steps.ci.openshift.org/job?org=Azure&repo=ARO-HCP&branch=main&test=image-updater-images)<br>[periodic-images](https://steps.ci.openshift.org/job?org=Azure&repo=ARO-HCP&branch=main&test=periodic-images) |
+| **Purpose** | Builds and validates container images for the project. The standard `images` job builds the main service images, while `image-updater-images` builds the image updater tooling variant, and `periodic-images` builds the images used by periodic jobs. |
+
+---
+
+#### Frontend Simulation
+
+| Property | Value |
+|----------|-------|
+| **Job Name** | [`pull-ci-Azure-ARO-HCP-main-frontend-simulation`](https://prow.ci.openshift.org/?job=pull-ci-Azure-ARO-HCP-main-frontend-simulation) |
+| **Trigger** | `/test frontend-simulation` |
+| **Status** | Always runs (required) |
+| **Cluster** | build10 |
+| **Step Registry** | [frontend-simulation](https://steps.ci.openshift.org/job?org=Azure&repo=ARO-HCP&branch=main&test=frontend-simulation) |
+| **Purpose** | Simulates and tests the frontend service functionality. This job runs on a cluster with nested Podman capability to support containerized testing scenarios. |
+
+---
+
+#### E2E Parallel
+
+| Property | Value |
+|----------|-------|
+| **Job Name** | [`pull-ci-Azure-ARO-HCP-main-e2e-parallel`](https://prow.ci.openshift.org/?job=pull-ci-Azure-ARO-HCP-main-e2e-parallel) |
+| **Trigger** | `/test e2e-parallel` |
+| **Status** | Always runs, but optional (does not block merge) |
+| **Environment** | Dev (westus3) |
+| **Cluster** | build07 |
+| **Step Registry** | [e2e-parallel](https://steps.ci.openshift.org/job?org=Azure&repo=ARO-HCP&branch=main&test=e2e-parallel) |
+| **Purpose** | Runs end-to-end tests in parallel mode against the dev environment. This job always runs on PRs but is marked optional, meaning failures won't block the PR from merging. |
+
+---
+
+#### Environment-Specific E2E Tests
+
+These optional jobs allow testing against specific Azure environments before merging changes.
```

Comment:
```suggestion
These optional jobs allow testing against specific Azure environments before merging changes in [ARO HCP E2E test case code](https://github.com/Azure/ARO-HCP/tree/main/test). Using these jobs to validate changes in code of ARO HCP RP or ARO HCP infrastructure deployment is not possible, because changes in these components won't be available in Azure environments before a merge of given pull request.
```